addons/map_gestures/functions/fnc_getProximityPlayers.sqf

```diff
@@ -4,22 +4,38 @@
  * Returns all players in a given range and in the units vehicle.
  *
  * Arguments:
- * 0: Unit <OBJECT>
+ * 0: Positions <ARRY<OBJECT, ARRAY>, BOOL>
  * 1: Range <NUMBER>
  *
  * Return Value:
- * All units in proximity <ARRAY>
+ * All units in proximity <ARRAY<OBJECT>>
  *
  * Example:
- * [player, 7] call ace_map_gestures_fnc_getProximityPlayers
+ * [[player], 7] call ace_map_gestures_fnc_getProximityPlayers
  *
  * Public: No
  */
 
-params ["_unit", "_range"];
+params ["_positions", "_range"];
 
-private _proximityPlayers = _unit nearEntities [["CAMAnBase"], _range];
-_proximityPlayers deleteAt (_proximityPlayers find _unit);
-_proximityPlayers append (crew vehicle _unit);
+private _proximityPlayers = [];
+if (_positions isEqualType objNull || { _positions isEqualTypeArray [0, 0, 0] } || { _positions isEqualTypeArray [0, 0] }) then {
+    _positions = [_positions];
```

Comment:
all the calls to this should be using correct params so I think we could drop this